src/plugins/embeddable/public/lib/embeddable_placement/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export {
```

Comment:
Question about this registry, should this be located in embeddable or dashboard plugin? How coupled is the registry to dashboard?

Reply:
Discussed offline with @nreese  and @ThomThomson and we will move the registry to the Dashboard plugin. Any plugins that need to register a panel placement strategy will need to import the registry functions from dashboard.